.github/workflows/release.yml

```diff
@@ -58,7 +58,7 @@ jobs:
   release-test-pypi:
     needs: [build]
     runs-on: ubuntu-latest
-    # if: github.event.inputs.release-test-pypi == 'true'
+    if: github.event.inputs.release-test-pypi == 'true'
```

Comment:
What does this line do?

Reply:
when run manually you can input wether to release or simply build the artifacts. This is good for debugging purposes.